### PR TITLE
Changed numbering to `-`

### PR DESCRIPTION
**Pull request by Ki-er**
Changed numbers to `-` to match other lists on the GitHub Platform while also making it easier to add entries with a simple return

### CODE REVIEW

**Review by CrimsonTome — CHANGES_REQUESTED** [review]
Looking good, would you be happy to do the same in other files in this repo? as the readme does not contain every resource.

File: Ethics_Law_Professionalism.md
Comment (CrimsonTome) [inline Ethics_Law_Professionalism.md]: is this part of a link??

Comment (Ki-er) [reply Ethics_Law_Professionalism.md]: https://github.com/FreesideHull/StudentResources/blob/main/Ethics_Law_Professionalism.md#professionalism--standards- - This is within the repo currently so im not sure 😆

**Review by CrimsonTome — CHANGES_REQUESTED** [review]
Thank you so much for the reformatting! regarding the comment I made, if this is just a stray part of a link could it be removed?

File: Ethics_Law_Professionalism.md
Comment (CrimsonTome) [reply Ethics_Law_Professionalism.md]: I'd say remove the `guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/)` part since it looks like it would have been a link but is not complete

**Comment by Ki-er** [comment]
Changes have been made

**Review by CrimsonTome — APPROVED** [review]
TY!

**Comment by CrimsonTome** [comment]
Because of the large (well aesthetic) change to formatting I will wait for one other review before merging this PR.

**Review by lgibson02 — APPROVED** [review]

**Pull request merged**
